src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,4 @@
+$cell-size: 100px;
+$cell-count: 31;
+$distans-between-cell: 1px;
```

Comment:
There is a typo in the variable name: 'distans-between-cell' should be 'distance-between-cell' 🧐